webpack.config.js

```diff
@@ -17,7 +17,7 @@
  */
 
 // Depends
-var _ = require('lodash');
+var merge = require('webpack-merge');
```

Comment:
Yo! I'am not sure about this line. 
Why you bring another one package for simple merge function like 
```
/**
 * Simple is object check.
 * @param  {Object} item
 * @return {boolean}
 */
const isObject = item => {
  return (item && typeof item === 'object' && !Array.isArray(item) && item !== null);
};

/**
 * Deep merge two objects.
 * @param  {object} target [description]
 * @param  {object} source [description]
 * @return {object}        [description]
 */
const mergeDeep = (target, source) => {
  if (isObject(target) && isObject(source)) {
    for (const key in source) {
      if (isObject(source[key])) {
        if (!target[key]) Object.assign(target, { [key]: {} });
        mergeDeep(target[key], source[key]);
      } else {
        Object.assign(target, { [key]: source[key] });
      }
    }
  }

  return target;
};
```

Hope it's help